pepper-apis/src/main/java/org/broadinstitute/ddp/model/event/CopyAnswerEventAction.java

```diff
@@ -0,0 +1,108 @@
+package org.broadinstitute.ddp.model.event;
+
+import java.util.List;
+
+import com.typesafe.config.ConfigFactory;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.broadinstitute.ddp.db.AnswerDao;
+import org.broadinstitute.ddp.db.dao.JdbiActivityInstance;
+import org.broadinstitute.ddp.db.dao.JdbiLanguageCode;
+import org.broadinstitute.ddp.db.dto.EventConfigurationDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.activity.instance.answer.Answer;
+import org.broadinstitute.ddp.model.activity.instance.answer.BoolAnswer;
+import org.broadinstitute.ddp.model.activity.instance.answer.TextAnswer;
+import org.broadinstitute.ddp.model.activity.types.EventTriggerType;
+import org.broadinstitute.ddp.pex.PexInterpreter;
+import org.jdbi.v3.core.Handle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CopyAnswerEventAction extends EventAction {
```

Comment:
FYI I have just merged in some new changes to `CopyAnswerService` for age-up (#7). Please help move those over to this new action class. Thanks!

Reply:
On it